### PR TITLE
pythonPackages.sievelib: init at 1.1.1

### PR DESCRIPTION
**Pull request by leenaars**
###### Motivation for this change

Useful library that implements various RFC's.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/sievelib/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{ stdenv, lib, isPy3k, buildPythonPackage, fetchPypi, fetchpatch, mock
```
Comment (costrouc) [inline pkgs/development/python-modules/sievelib/default.nix]: `isPy3k` and `stdenv` are not used in derivation

**Review by costrouc — CHANGES_REQUESTED** [review]

**Comment by leenaars** [comment]
@costrouc : You were right, those were remnants that could be removed. They are gone now.

**Comment by costrouc** [comment]
@GrahamcOfBorg build python3Packages.sievelib python2Packages.sievelib

**Comment by leenaars** [comment]
@costrouc: Are the edits I made okay?

**Comment by costrouc** [comment]
Yeah it looks great to me. I don't have permissions to merge PRs. @FRidh would be the one to do that.

File: pkgs/development/python-modules/sievelib/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+A library written in Python that implements RFC 5228 (Sieve : An Email
+Filtering Language) and RFC 5804 (ManageSieve : A Protocol for
+Remotely Managing Sieve Scripts), as well as the following extensions:
+
+ * Copying Without Side Effects (RFC 3894)
+ * Body (RFC 5173)
+ * Date and Index (RFC 5260)
+ * Vacation (RFC 5230)
+ * Imap4flags (RFC 5232)
+'';
```
Comment (dotlambda) [inline pkgs/development/python-modules/sievelib/default.nix]: Please fix the indentation.

File: pkgs/development/python-modules/sievelib/default.nix [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  version = "1.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1sl1fnwr5jdacrrnq2rvzh4vv1dyxd3x31vnqga36gj8h546h7mz";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/tonioo/sievelib/commit/1deef0e2bf039a0e817ea6f19aaf1947dc9fafbc.patch";
```
Comment (dotlambda) [inline pkgs/development/python-modules/sievelib/default.nix]:
```suggestion
      url = "${meta.homepage}/commit/1deef0e2bf039a0e817ea6f19aaf1947dc9fafbc.patch";
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by c0bw3b** [comment]
@GrahamcOfBorg build python3Packages.sievelib python2Packages.sievelib

**Pull request merged**